src/main/java/uk/ac/ebi/ena/webin/cli/context/transcriptome/TranscriptomeManifestReader.java

```diff
@@ -90,7 +95,9 @@ public TranscriptomeManifestReader(
 					.meta().optional().name( Field.TPA          ).desc( Description.TPA ).processor( CVFieldProcessor.CV_BOOLEAN ).and()
 					.meta().optional().name( Field.AUTHORS      ).desc( Description.AUTHORS ).processor(new AuthorProcessor()).and()
 					.meta().optional().name( Field.ADDRESS      ).desc( Description.ADDRESS ).and()
-						.meta().required().name( Field.ASSEMBLY_TYPE    ).desc( Description.ASSEMBLY_TYPE    ).processor( new CVFieldProcessor( CV_ASSEMBLY_TYPE ) )
+					.meta().required().name( Field.ASSEMBLY_TYPE    ).desc( Description.ASSEMBLY_TYPE    ).processor( new CVFieldProcessor( CV_ASSEMBLY_TYPE ) ).and()
+					.meta().optional().name(ReadsManifestReader.Field.SUBMISSION_TOOL).desc(ReadsManifestReader.Description.SUBMISSION_TOOL).and()
+					.meta().optional().name(ReadsManifestReader.Field.SUBMISSION_TOOL_VERSION).desc(ReadsManifestReader.Description.SUBMISSION_TOOL_VERSION)
```

Comment:
Should not point to ReadsManifestReader Field/Description. Should use the fields from this class. Ot we could add them into a common class.